api/views.py

```diff
@@ -14,6 +15,21 @@
 )
 
 
+class QueryMetadata(SimpleMetadata):
+    """Additionally include info about query parameters."""
+
+    def determine_metadata(self, request, view):
+        print('doing something')
```

Comment:
Good thing I don't have a habit of using swear words in debug-printing anymore :-)